lib/services/users.pm

```diff
@@ -165,7 +168,12 @@ sub full_users_check {
     }
     else {
         #swtich to new added user then switch back
-        switch_users;
+        if (check_var('ARCH', 's390x')) {
```

Comment:
Any known issues/bugs we can't switch user like other platforms? seems we add a workaround here

Reply:
See PR description, comments#2 of https://bugzilla.suse.com/show_bug.cgi?id=1185943

Reply:
Yup, got it.